src/dvr/dvr_rec.c

```diff
@@ -124,8 +128,21 @@ dvr_rec_subscribe(dvr_entry_t *de)
     }
   }
 
-  if(stat(de->de_config->dvr_storage, &st) || !S_ISDIR(st.st_mode)) {
-    tvherror(LS_DVR, "the directory '%s' is not accessible", de->de_config->dvr_storage);
+  errSaved = 0;
+  statRet = stat(de->de_config->dvr_storage, &st);
+  errSaved = errno;
```

Comment:
Where it can be easily done, I usually save the ERRNO to a variable immediately after the operation in question.  I do this because I have personally encountered consecutive errors masking the original cause of the error.

However, if this is a mandatory requirement, I will remove the variable.

Reply:
In this case there is no chance for consecutive errors as there is no operation accessing errno in between. I'm a little hesitant as such copying always uses additional resources, and I am not sure if the compiler would optimize that out again in this case (which would be alright again).

We should attempt to match the coding style of the existing project, so please have a look and then let's do it exactly like it's done everywhere else. We can change it everywhere in case we don't like it in another PR

Reply:
I will remove the variable.